src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,18 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    private static final int MAX_FIGURES = 6;
 
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        Figure[] figures = new Figure[MAX_FIGURES];
+        for (int i = 0; i < figures.length; ++i) {
+            if (i < 3) {
+                figureSupplier.getRandomFigure().printInfo();
+                continue;
+            }
+            figureSupplier.getDefaultFigure().printInfo();
+        }
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/63095184/9950d201-eaaf-4e36-b739-86a7457dd65d)

Reply:
Is my version of this part bad ? Why ?